local/default/env

```diff
@@ -1,3 +0,0 @@
-export DATABASE_URL='postgres://postgres:password@localhost:5432'
```

Comment:
Please add this file back.

Reply:
Ok, I ll make changes. there is always mess when I do rebase :D